### DISCUSSION TITLE
Week 9

### DISCUSSION THREAD

**Discussion by ColinFay** [ColinFay/ur-first-5k#371] [General]
Hey y'all! 

Welcome to week 9 :) 

We still have two more weeks to go in the program! 
Hope everyone is still enjoying it.

Here is this week menu: 

```
── Day 1 
→ Rest
Rest day.

── Day 2 
→ W09D2-Intervals
• Warm up, 5 to 10 minutes.
• Run, steady pace, 5 minutes. Walk, 2 minutes. Repeat 4 times.
• Cool down, 5 to 10 minutes.
• Stretch.

── Day 3 
→ Rest
Rest day.

── Day 4 
→ W9D4-Steady Run
• Run, easy pace, 10 minutes.
• Run, steady pace, 10 minutes.
• Run, easy pace, 10 minutes.
• Cool down, 5 to 10 minutes.
• Stretch.

── Day 5 
→ Rest
Rest day.

── Day 6 
→ Rest
Optional: Swap Sunday's run to today.

── Day 7 
→ W09D7-Easy Run
• Run, easy pace, 30 to 40 minutes.
• Cool down, 5 to 10 minutes.
• Stretch.
```

Feel free to use this discussion thread if you have any question, suggestion, or if you just want to share your experience :) 

Enjoy ! 

Colin

**Comment by MaelaKloareg** [comment]
No cross training ? How strange, I'm getting used to my 4 assignments per week ;-)
Yes, I'm enjoying it more and more... I was already enjoying it at the beginning, but I feel like I might get addicted now ! :-)

**Reply by ColinFay** [reply]
Nop, not this week but of course you can still do it if you want/need to ;) 

Glad to hear you're enjoying the plan... and welcome to the running addict family :D 🤘
